doc/newshell.md

```diff
@@ -2,7 +2,7 @@
 
 Rizin has moved away from the default way of parsing radare2 commands and the
 way commands were handled there. It enables by default what is called in r2
-`cfg.newshell`, which enables a generated parser that parses rizin commands and
+`cfg.rzshell`, which enables a generated parser that parses rizin commands and
```

Comment:
i would prefer to have this called `cfg.oldshell` and just call `newshell` as `shell`

Reply:
I think it doesn't matter since after everything will be ported it will be removed anyway.

Reply:
Ok I have replaced this with `cfg.oldshell` so newshell has no more a name . It is just *the shell* :P